api/validations/order.validation.ts

```diff
@@ -0,0 +1,14 @@
+import { ORDER_STATUSES } from "../types";
+import { isISODate } from "../utils";
+import { z } from "zod";
+
+export const OrderValidation = z.object({
+  member: z.string().min(1),

```

Comment:
Insert entire member

Reply:
Do not insert the member, as the member's first and last name is still requested within the order and shipments.

Reply:
Add //TODO